ql/src/java/org/apache/hadoop/hive/ql/io/CombineHiveRecordReader.java

```diff
@@ -113,7 +113,7 @@ private PartitionDesc extractSinglePartSpec(CombineHiveInputSplit hsplit) throws
     for (Path path : hsplit.getPaths()) {
       PartitionDesc otherPart = HiveFileFormatUtils.getFromPathRecursively(
           pathToPartInfo, path, cache);
-      LOG.debug("Found spec for " + path + " " + otherPart + " from " + pathToPartInfo);
+      LOG.debug("Found spec for {} {} from {}", path, otherPart, pathToPartInfo);
```

Comment:
Can we have this inside the block?
 if (LOG.isDebugEnabled()) {
}

Reply:
yes, we can
I'm assuming that logger libraries check this anyway first, but yeah, maybe an early-check makes sense (I just tried to avoid boilerplate)

Reply:
Have done this in the past using {} so I am fine with either solution --  Lets just keep this consistent with HIVE-16150

Reply:
HIVE-16150 was doing with guards + concatenation which is 3 lines of code
here I preferred `{}` which is only 1 line and looks cleaner
so from this point of view, this is not consistent with HIVE-16150, but this is the better way to do it

Reply:
Sounds good! Thanks @abstractdog